Code/Tools/FBuild/FBuildCore/Graph/LinkerNode.cpp

```diff
@@ -711,17 +711,17 @@ void LinkerNode::GetAssemblyResourceFiles( Args & fullArgs, const AString & pre,
 
             if ( IsStartOfLinkerArg_MSVC( token, "OPT" ) )
             {
-                if ( token.FindI( "REF" ) )
+                if ( token.FindI( ":REF" ) )
```

Comment:
Options can also be comma delimited, i.e. /OPT:REF,ICF so this would break those cases.